packages/teleport/src/Console/DocumentNodes/ClusterSelector/ClusterSelector.tsx

```diff
@@ -104,111 +103,23 @@ function filterOptions(value = '', options: Option[] = []) {
 
 type Option = { value: string; label: string };
 
-const StyledSelect = styled(Box)(
+const StyledSelect = styled(DarkStyledSelect)(
   ({ theme }) => `
-  .react-select__control,
-  .react-select__control--is-focused {
-    border-color: #FFF;
-    height: 34px;
-    min-height: 34px;
+  .react-select-container {
+    background: ${theme.colors.primary.light};
   }
 
-  .react-select__option {
-    padding: 4px 12px;
-  }
-  .react-select__option--is-focused,
-  .react-select__option--is-focused:active {
-    background-color: ${theme.colors.grey[50]};
-  }
-
-  .react-select__menu {
-    margin-top: 0px;
-    font-size: 14px;
-  }
-
-  react-select__menu-list {
-  }
-
-  .react-select__indicator-separator {
-    display: none;
-  }
-
-  .react-select__value-container{
-    height: 30px;
-    padding: 0 8px;
+  .react-select__single-value {
```

Comment:
lets add back the `color: white`, i did a side by side test, and the white looks better on the very dark bg